app/src/main/java/org/oppia/app/databinding/GuidelineBindingAdapters.java

```diff
@@ -0,0 +1,17 @@
+package org.oppia.app.databinding;
+
+import androidx.annotation.NonNull;
+import androidx.constraintlayout.widget.Guideline;
+import androidx.constraintlayout.widget.ConstraintLayout;
```

Comment:
Okay!

Reply:
When hitting reformats also do the optimize import which will correct Import Sequence and if there are any unused imports it will remove automatically. 
I suggest using Macros you an do reformat and optimize all together, no repetitive work. Check this wiki page [here](https://github.com/oppia/oppia-android/wiki/Ktlint-Guide#macros)

Reply:
Wow! This just made my life so much easier! Thanks :)